src/ui/MainWindow.cc

```diff
@@ -183,8 +183,10 @@ MainWindow::MainWindow(QSplashScreen* splashScreen)
     // Qt 4 on Ubuntu does place the native menubar correctly so on Linux we revert back to in-window menu bar.
     // TODO: Check that this is still necessary on Qt5 on Ubuntu
 #ifdef Q_OS_LINUX
+#ifndef __android__
     menuBar()->setNativeMenuBar(false);
 #endif
+#endif
     

```

Comment:
Well, for the time being, it's the only way to access the rest of the UI. Otherwise you can only access the four things the four main tool bar buttons allow you to access. But yes, it should go away once we have a proper, mobile style menu.